src/Data/Function.purs

```diff
@@ -26,6 +26,12 @@ flip f b a = f a b
 -- | ```purescript
 -- | const 1 "hello" = 1
 -- | ```
+-- |
+-- | Can also be thought of as creating a function that ignores its argument:
```

Comment:
I don't know if this style is used elsewhere, but I prefer using complete sentences, either be prefixing with "It can also be...", or referring to the name again.

Reply:
I was just basing this on the previous line which is also a fragment:
> Returns its first argument and ignores its second.